src/index.js

```diff
@@ -0,0 +1,50 @@
+import './style.css';
+import './style.scss';
+import 'bootstrap';
+
+const toDolist = document.getElementById('toDoList');
+const addBtn = document.getElementById('addBtn');
+const inputs = document.getElementsByClassName('forminput');
+let listArray = [];
+
+function addList() {
+  const listarr = {
+    name: toDolist.value,
+  };
+  listArray.push(listarr);
+}
+
+function displayData() {
+  let trs = '';
+  for (let i = 0; i < listArray.length; i += 1) {
+    trs += `<tr>
+     <td><input type="checkbox"></td>
+    <td class="float-left">${listArray[i].name}</td>
+    <td><button onclick="deleteList(${i})" class="float-right"><i class="fa fa-ellipsis-v"></i></button></td>
+    </tr>`;
+  }
+  document.getElementById('tableBody').innerHTML = trs;
+}
```

Comment:
- [ ] You have a function that renders the array. That's good. However, you need to ensure your design is similar to what we have on the template shared with you.

  The template | Your design
  :---:|:---:
  ![karam-todo-template](https://user-images.githubusercontent.com/36542986/160043288-752c86db-b05e-4c10-b6c3-3e5550698979.jpg) | ![karam-todo-implement](https://user-images.githubusercontent.com/36542986/160043283-304da57e-98f0-499c-8b16-3c51487ed8f0.jpg)